client/src/test/java/io/pravega/client/stream/StreamCutTest.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.stream;
+
+import com.google.common.collect.ImmutableMap;
+import io.pravega.client.segment.impl.Segment;
+import io.pravega.client.stream.impl.StreamCutImpl;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import lombok.Cleanup;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class StreamCutTest {
```

Comment:
I'm a bit surprised that we don't have a class like this at the moment to unit test stream cut functionality. Where is all the testing on stream cuts?

Reply:
- https://github.com/pravega/pravega/blob/master/test/integration/src/test/java/io/pravega/test/integration/StreamCutsTest.java
- StreamCuts when used inside a ReaderGroup have their separate tests.